webcompat/api/uploads.py

```diff
@@ -82,7 +83,15 @@ def save(self):
         dest_dir = os.path.dirname(file_dest)
         if not os.path.exists(dest_dir):
             os.makedirs(dest_dir)
-        self.image_object.save(file_dest)
+        # Optimize further the image compression for these formats
+        if self.image_object.format in ['JPEG', 'JPG', 'JPE', 'PNG']:
```

Comment:
Hm, I wonder if (in a future patch) we can somehow merge this list and the `ALLOWED_FORMATS`. I'm sure there's a nicer way to do that.

(I also see something about BMP compression here: http://pillow.readthedocs.org/en/3.1.x/handbook/image-file-formats.html, but I doubt we'll be getting many of those files anyways).